tasks/main.yml

```diff
@@ -33,6 +40,7 @@
     name: postgresql.service
     state: restarted
   when: gen_locale.changed
+  tags: travis-exempt
```

Comment:
This doesn’t appear to be Travis specific, but systemd specific. I think this would be best to be more generic than “Travis”